src/utils.ts

```diff
@@ -17,7 +17,8 @@ import RNSError, { NO_ACCOUNTS_TO_SIGN } from './errors';
  * true if method exists, false if not
  */
 export const hasMethod = async (web3: Web3, contractAddress: string, signatureHash: string) => {
-  const code = await web3.eth.getCode(contractAddress);
+  const namespace = web3.eth || web3;
```

Comment:
Instead of `web3.eth || web3` in multiple places, I would recommend that you do this right after running the constructor for rsk3.js.

```
const rsk3 = new Rsk3(...);
rsk3.eth = rsk3
```

... and then you don't have to remember to do this each time.

By way of an example, see:

https://github.com/rsksmart/json-rpc-web3js/blob/4045638/smoke-test.js#L14-L21

and 

https://github.com/rsksmart/json-rpc-web3js/blob/4045638/smoke-test.js#L41-42